packages/runtime-dom/src/nodeOps.ts

```diff
@@ -27,6 +27,19 @@ export const nodeOps: Omit<RendererOptions<Node, Element>, 'patchProp'> = {
       ;(el as HTMLSelectElement).setAttribute('multiple', props.multiple)
     }
 
+    if (tag === 'option' && props && props.selected) {
+      el.setAttribute('selected', '')
+    }
+
+    if (tag === 'input' && props) {
+      if (props.value) {
```

Comment:
Should be (props.value != null), otherwise it gonna skip value 0.

Reply:
Good catch, thx!